conan/cli/commands/graph.py

```diff
@@ -166,7 +169,8 @@ def graph_info(conan_api, parser, subparser, *args):
                                                       clean=args.lockfile_clean)
         conan_api.lockfile.save_lockfile(lockfile, args.lockfile_out, os.getcwd())
         if args.deployer:
-            base_folder = os.getcwd()
+            base_folder = args.output_deployer or os.getcwd()
+            mkdir(base_folder)
```

Comment:
There is already a ``mkdir()`` for this inside ``do_deploys``, this shouldn't be necessary?